src/hooks/useRemoteConnection.ts

```
@@ -54,23 +58,39 @@ export const useRemoteConnection = () => {
             socket.onerror = () => {
                 socket.close();
             };
+            socket.onmessage = (event) => {
+                try {
+                    if (event.data instanceof ArrayBuffer) {
+                        if (isMounted) {
+                            listenersRef.current.forEach(l => l({ type: 'mirror-frame-bin', data: event.data }));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)